cli/tests/integrations/test_config.py

```diff
@@ -72,7 +72,7 @@ def test_version():
 
 
 def test_list_property(env):
-    stdout = b"""core.dcos_url=http://172.17.8.101
+    stdout = b"""core.dcos_url=http://change.dcos.url
```

Comment:
Well, it passes for py27 but not for py34. Because of the `env` parameter, this test is running against `test/data/dcos.toml` and not against `/tmp/teamcity_dcos.toml` (TC's DCOS_CONFIG).